YourScheduler/YourScheduler.Infrastructure/Entities/TeamRoleFlags.cs

```diff
@@ -23,5 +17,25 @@ public class TeamRoleFlags
         public bool CanEditTeamEvent { get; set; } = false;
         public bool CanRemoveTeamEvent { get; set; } = false;
         public bool CanSendEmailToTeam { get;set; } = false;
+
+        public TeamRoleFlags(int teamRoleId, bool canRemoveTeamMember, bool canAddTeamMember, bool canAddTeamRole, bool canEditTeamRole, bool canRemoveTeamRole, bool canEditRoleFlags, bool canEditTeamPhoto, bool canEditDescription, bool canEditTeamMessage, bool canEditTeamName, bool canAddTeamEvent, bool canEditTeamEvent, bool canRemoveTeamEvent, bool canSendEmailToTeam)
```

Comment:
Panie, co to za konstruktor tutaj? O co tutaj chodzi Panie?

Reply:
Trochę z lenistwa, łatwiej mi tworzyć TeamRole w testach, naprawię.

Reply:
Naprawione w ostatnim commitcie.